s3storageprovider/src/main/java/org/duracloud/s3storage/S3StorageProvider.java

```diff
@@ -70,6 +75,11 @@
     private static final StorageClass DEFAULT_STORAGE_CLASS =
         StorageClass.Standard;
 
+    private static final String UTF_8 = StandardCharsets.UTF_8.name();
+    
+    private static final String HEADER_VALUE_PREFIX = UTF_8 + "' '";
```

Comment:
Good catch.  It looked like there was a space to me, but on closer examination, you're right - no space.